<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>database_window</class>
 <widget class="QWidget" name="database_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#database_window {
background-color: rgb(97, 97, 97);
color: black;
}
</string>
  </property>
  <widget class="QLabel" name="label_main_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>769</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_main_text {
        font-size: 25px;
        border: 3px solid rgb(179, 0, 0);
        border-radius: 15px;
        background-color: rgb(171, 171, 171);
		color:black;
}

QLabel#label_main_text::hover {
        background-color: rgb(223, 223, 223);
}</string>
   </property>
   <property name="text">
    <string>Приложение для взаимодействия с базой данных</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>671</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="title">
    <string>Подключение к базе данных</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>651</width>
      <height>229</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_ip_database">
         <property name="styleSheet">
          <string notr="true">QLabel#label_ip_database {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Адрес расположения базы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ip_database">
         <property name="styleSheet">
          <string notr="true">QLineEdit#lineEdit_ip_database {
	background-color: white;
	color: black;
}</string>
         </property>
         <property name="placeholderText">
          <string>Если база на локальной машине, введите localhost.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_port">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel#label_port {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Порт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_port">
         <property name="styleSheet">
          <string notr="true">QLineEdit#lineEdit_port {
	background-color: white;
	color: black;
}</string>
         </property>
         <property name="placeholderText">
          <string>Порт сервера.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_user">
         <property name="styleSheet">
          <string notr="true">QLabel#label_user {
	color: white;
}
</string>
         </property>
         <property name="text">
          <string>Имя пользователя:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_user">
         <property name="styleSheet">
          <string notr="true">QLineEdit#lineEdit_user {
	background-color: white;
	color: black;
}</string>
         </property>
         <property name="placeholderText">
          <string>Имя пользователя. Например: postgres.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_password">
         <property name="styleSheet">
          <string notr="true">QLabel#label_password {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_password">
         <property name="styleSheet">
          <string notr="true">QLineEdit#lineEdit_password {
	background-color: white;
	color: black;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
         <property name="placeholderText">
          <string>Пароль для доступа к серверу.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::EditFind"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_database_name">
         <property name="styleSheet">
          <string notr="true">QLabel#label_database_name {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Имя базы данных:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_database_name">
         <property name="styleSheet">
          <string notr="true">QLineEdit#lineEdit_database_name {
	background-color: white;
	color: black;
}</string>
         </property>
         <property name="placeholderText">
          <string>Имя вашей базы данных.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButton_connect">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#pushButton_connect {
	background-color: rgb(159, 255, 152);
	color: black;
}</string>
         </property>
         <property name="text">
          <string>Подключиться</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_remember_me">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox::indicator {
    width: 16px;
    height: 16px;
}

QCheckBox::indicator:unchecked {
	background-color: white;
}</string>
         </property>
         <property name="text">
          <string>Запомнить меня</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
